model/dataProvider/SyncPackageDataProviderServiceInterface.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019  (original work) Open Assessment Technologies SA;
+ *
+ * @author Oleksandr Zagovorychev <zagovorichev@gmail.com>
+ */
+
+namespace oat\taoSyncClient\model\dataProvider;
+
+
+interface SyncPackageDataProviderServiceInterface extends SyncPackageDataProviderInterface
```

Comment:
I'm not sure we need this interface at all. Most probably we will not need to have another implementation. And I cannot see a use case when we would need to call `getProvider` method from outside of `SyncClientDataProviderService`.

Reply:
As was discussed - I left service interface but made the method getProvider as private